src/main/java/net/sourceforge/plantuml/activitydiagram3/ftile/Swimlanes.java

```diff
@@ -247,11 +251,16 @@ public final void drawU(UGraphic ug) {
 	private void drawGtile(UGraphic ug) {
 		TextBlock full = root.createGtile(skinParam, ug.getStringBounder());
 
+		final Style style = skinParam.getCurrentStyleBuilder().getMergedStyle(
+			StyleSignatureBasic.of(SName.root, SName.element, SName.activityDiagram, SName.goto_));
+		final HColor gotoColor = style.value(PName.LineColor).asColor(skinParam.getIHtmlColorSet());
+		final boolean isDebug = true;
+
 		ug = new UGraphicForSnake(ug);
 		if (swimlanes().size() > 1) {
 			drawWhenSwimlanes(ug, full);
 		} else {
-			full = new TextBlockInterceptorUDrawable(full);
+			full = new TextBlockInterceptorUDrawable(full, gotoColor, isDebug);
```

Comment:
@arnaudroques:

- Is it necessary to have redundancy for `Gtile` also?

Reply:
> * Is it necessary to have redundancy for `Gtile` also?

Yes, sorry about that!
`Gtile` is stale right now, so there are some code duplication...